letslambda.json

```diff
@@ -58,7 +145,12 @@
         "LambdaExecutionRole": {
             "Type": "AWS::IAM::Role",
             "Properties": {
-                "ManagedPolicyArns": [ {"Ref": "LetsLambdaManagedPolicy"} ],
+                "ManagedPolicyArns": [
+                    {"Ref": "LetsLambdaManagedPolicy"},
+                    {"Ref": "LetsLambdaS3WriteManagedPolicy"},
+                    {"Ref": "LetsLambdaS3ReadManagedPolicy"},
+                    {"Ref": "LetsLambdaKmsKeyManagedPolicy"}
```

Comment:
I've found in the past that creating a policy per logical action is reasonable, and then enlisting policies to roles as necessary. That allows a user (with appropriate permission) to assume the role and debug the permissions really easily.